internal/services/costmanagement/anomaly_alert_resource.go

```diff
@@ -117,6 +124,13 @@ func (r AnomalyAlertResource) Create() sdk.ResourceFunc {
 			schedule.SetEndDateAsTime(time.Now().AddDate(1, 0, 0))
 			schedule.SetStartDateAsTime(time.Now())
 
+			var senderEmail string
+			if v, ok := metadata.ResourceData.GetOk("notification_email"); ok {
+				senderEmail = v.(string)
+			} else {
+				senderEmail = (*emailAddresses)[0]
+			}
+
```

Comment:
Could we update this to 
```suggestion
			notificationEmail := (*emailAddresses)[0]
			if v, ok := metadata.ResourceData.GetOk("notification_email"); ok {
				notificationEmail = v.(string)
			}
```